settings.py

```diff
@@ -0,0 +1,62 @@
+import os
+from distutils.util import strtobool as _strtobool
+
+
+def strtobool(value):
+    try:
+        return bool(_strtobool(value))
+    except ValueError:
+        return False
+
+
+def env(variable, fallback_value=None):
+    if os.environ.get('SUPERDESK_USE_DEFAULTS'):
```

Comment:
I would avoid having the same prefix as superdesk, during deploy it may confuse. 
We can use `VIDEOSERVER_USE_DEFAULTS`.
Does it makes sense?